src/app/shared/components/List/index.tsx

```diff
@@ -283,14 +283,14 @@ ListProps<T, F>): JSX.Element {
                 />
               ))}
             </InfiniteScroll>
-          </>
-        ) : (
-          <>{loader}</>
-        )}
-
-        {/* Click outside of the list to unselect all items */}
-        {items.length > 0 && <div className="h-full w-full py-6" onClick={unselectAllItems} />}
-      </div>
+            {items.length > 0 && <div className="h-full w-full py-6" onClick={unselectAllItems} />}
```

Comment:
What is the reason for moving this? Don't forget to move the comment that refers to it.